### DISCUSSION TITLE
Introduction and Discussion about Remote Sessions

### DISCUSSION THREAD

**Discussion by PatrickHeneise** [cyprusjs/CyprusJS#36] [Proposals]
We're all tired of "remote beers" and "social zooms", so instead we'd like to get together for an actual challenge and to learn something new.

In this first session we'll discuss how this could look like, if/how we'll stream and code together, and which projects we'll support.

**Comment by PatrickHeneise** [comment]
Project proposal: "Rarely - A platform for allowing 1+ billion differently abled people to share their ideas and lived experiences and improve life." https://github.com/rarely-org/rarely

**Reply by alielgin** [reply]
I love the idea

**Comment by NVerban** [comment]
Great idea! What do you have in mind for tech stack / framework?

**Reply by PatrickHeneise** [reply]
@steffen ^

**Comment by saitonakamura** [comment]
@NVerban there're discussions about that https://github.com/rarely-org/rarely/discussions

**Comment by steffen** [comment]
👋 Hi all! Thank you so much @PatrickHeneise for proposing Rarely! ❤️ 

I have started discussions on the tech stack here:
- [Technology stack (Frontend)](https://github.com/rarely-org/rarely/discussions/4)
- [Technology stack (Backend)](https://github.com/rarely-org/rarely/discussions/6)

My plan is to commit a scaffold project (likely only with the frontend using React Native for now as the backend part is still open for discussion) within the next days to the [repository](https://github.com/rarely-org/rarely) and create more specific next steps issues.

I'm hoping we can leverage [GitHub Codespaces](https://github.com/features/codespaces) to allow everyone to get right to the coding (and fun) part instead of dealing with the development environment setup. (It seems to work well enough with React Native + local mobile emulator.)
As Codespaces doesn't support outside collaborators yet, I'm happy to add anyone interested to the organization in order to make Codespaces accessible. I will share more about it.

Let me know if you have any questions! Thank you all again! 💖 Looking forward to the sessions! ✨

**Comment by steffen** [comment]
Cross-posting the event details for this Saturday's session (**December 4**):
https://www.linkedin.com/events/codeforagoodcausebycyprusjs6871136908521586688/about/
